events/c_2_project_and_community.toml

```diff
@@ -64,81 +66,39 @@ This is a track
 # - recommended lunch: 12:00 or 13:00
 # - recommended end:   17:00
 # - use 15m, 20m, 30m, 45m, or 60m session slots
-# - use 15m breaks, one in the morning, one or two in the afternoon
+
 [[timeslots]]
 startTime="09:00"
-speakers=["@speaker_name", "@other_speaker_name"]
+speakers=["@mishmosh"]
 title="Welcome standup discussion"
-description=""
+description="A brief introduction to the track"
 
 [[timeslots]]
-startTime="09:30"
-speakers=["@yourname"]
-title="Track Intro"
-description=""
+startTime="09:15"
+speakers=["@lidel"]
+title="Lightweight RFC Process"
+description="A walkthrough of the new RFC process and its motivations, updated specs repository, and how you can submit an RFC."
 
 [[timeslots]]
-startTime="10:00"
+startTime="09:45"
```

Comment:
@bmann is going to be attending (although I haven't confirmed he'll be there on Friday), and I've been talking with the fission folks about a rather spicy take on resource allocation that I think would be really interesting to dig into. Paraphrasing:

Why are we pouring resources into re-writing the same thing in x # of different languages? Wouldn't it be better to pool our efforts & write a kick-ass reference implementation in one language and write libraries in each language that wrap that impl?

I for one really want to hear the case laid out for such a thing. Maybe this is the right forum?